weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsPage.tsx

```diff
@@ -297,6 +321,15 @@ export const CallsTable: FC<{
     props.project
   );
 
+  useEffect(() => {
+    if (!calls?.loading && calls?.result != null && calls.result.length > 0) {
+      setIsRefSet(true);
```

Comment:
This feels a bit awkward to change if the ref is set based on the number of rows in the table. Perhaps these could be decoupled?

Reply:
Yes, its more like shouldShowExportButton, based on the number of rows in the table, let me change the name